docs/architecture-diagrams.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
```

Comment:
Fine to keep for now to make iteration easier, but let's plan on deleting this before merging to `#main`